modules/experimental-layers/src/gpu-screen-grid-layer/gpu-screen-grid-layer.js

```diff
@@ -248,6 +258,27 @@ export default class GPUScreenGridLayer extends Layer {
     attributeManager.invalidate('instanceCounts');
   }
 
+  _updateColorUniforms({oldProps, props}) {
+    const colorProps = [`minColor`, `maxColor`, `colorRange`, `colorDomain`];
+    if (
+      colorProps.some(key => {
+        return oldProps[key] !== props[key];
+      })
+    ) {
+      const shouldUseMinMax = this._shouldUseMinMax();
```

Comment:
Nit: I'd send in the props as argument to the `_shouldUseMinMax` method, just to make it a little clearer how it makes its decision.